benchmarks/benchmark_sqzb.py

```diff
@@ -243,6 +276,25 @@ def main(args: argparse.Namespace):
                              "Otherwise, we use Poisson process to synthesize "
                              "the request arrival times.")
     parser.add_argument("--seed", type=int, default=0)
+    parser.add_argument("--guided-json", action="store_true",
+                        help="Enable guided json")
+    parser.add_argument("--json-template", type=str, default=None,
+                        help="Path to guided json template. "
+                             "If not given, default json template is applied.")
```

Comment:
if json-template should always be given(as the line 298 says), lets just unify this two arguments to a single argument so that we can type less